features/configuration_files/masking_smells.feature

```diff
@@ -3,20 +3,6 @@ Feature: Masking smells using config files
   As a developer
   I want to mask some smells using config files
 
-  Scenario: empty config file is ignored
```

Comment:
Deleted as a suggestion in this PR - I can re-introduce it if we decide so.
I deleted it because to me, this feature doesn't make much sense. An empty config file shouldn't cause any errors / warnings at all.
E.g. for quick tests I sometimes use the pattern:

```Bash
echo "" > config.reek
reek ...
git co . # for git checkout .
```

And then this check is highly annoying. But it's not about my usage pattern, it's more about that we should warn if necessary, not just on all-the-things.
WDYT?